pkg/scheduler/framework/plugins/apienablement/api_enablement.go

```diff
@@ -55,8 +55,12 @@ func (p *APIEnablement) Filter(
 	cluster *clusterv1alpha1.Cluster,
 ) *framework.Result {
 	if !helper.IsAPIEnabled(cluster.Status.APIEnablements, bindingSpec.Resource.APIVersion, bindingSpec.Resource.Kind) {
-		klog.V(2).Infof("Cluster(%s) not fit as missing API(%s, kind=%s)", cluster.Name, bindingSpec.Resource.APIVersion, bindingSpec.Resource.Kind)
-		return framework.NewResult(framework.Unschedulable, "cluster(s) did not have the API resource")
+		if !bindingSpec.TargetContains(cluster.Name) {
```

Comment:
The core meaning of your PR is: If the binding has already been scheduled to a certain cluster, then we should ignore this `apienablement` plugin, right?

I have a question: If the AA service is no longer healthy, can it still create corresponding resources?

Reply:
Can we determine whether the cluster status needs to be updated based on the returned error? 

https://github.com/karmada-io/karmada/blob/a87ec2a61abadf9f102440b1f64cf338b2ee5518/pkg/controllers/status/cluster_status_controller.go#L233-L239

Only when the err description indicates that some APIs fail to be installed, the returned apiEnables will be updated to the cluster status.